.travis.yml

```diff
@@ -31,5 +31,4 @@ env:
   - TOXENV=pep8
   - TOXENV=check_commit_msg
   - TOXENV=py27-cover-master
-  - TOXENV=py27-cover-develop
```

Comment:
Do we need changes as part of this PR? I feel you need to rebase, otherwise changes look fine to me.